_posts/2020-09-07-sparklyr-flint/sparklyr_flint.Rmd

```diff
@@ -0,0 +1,227 @@
+---
+title: "Introducing sparklyr.flint: A time-series extension for sparklyr"
+description: |
+  We are pleased to announce sparklyr.flint, a sparklyr extension for analyizing time series at scale, is now available on CRAN.
+author:
+  - name: Yitao Li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+slug: sparklyr.flint-0.1.1
+date: 09-07-2020
+categories:
+  - R
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.png
+---
+
+In this blog post, we will showcase [`sparklyr.flint`](https://cran.r-project.org/web/packages/sparklyr.flint/index.html), a [`sparklyr`](https://sparklyr.ai) extension providing a simple and intuitive R interface to the [`Flint`](https://github.com/twosigma/flint) time series library.
+
+We will include a quick bird's eye view on `sparklyr` and `Flint` within the first two sections of this post and ensure readers unfamiliar with `sparklyr` or `Flint` can see both of them as essential building blocks for `sparklyr.flint`, and then without further ado, we will feature `sparklyr.flint`'s design philosophy, current state, example usages, and last but not least, its future directions as an open-source project in the subsequent sections.
+
+# Quick Intro to `sparklyr`
+
+`Sparklyr` is an open-source R interface that integrates the power of distributed computing from [Apache Spark](https://spark.apache.org) with the familiar syntaxes, tools, and paradigms for data transformation and data modelling in R. It allows data pipelines working well with non-distributed data in R to be easily transformed into analogous ones that can process large-scale, distributed data in Apache Spark.
```

Comment:
We prefer lowercase since CRAN is case-sensitive:

```suggestion
`sparklyr` is an open-source R interface that integrates the power of distributed computing from [Apache Spark](https://spark.apache.org) with the familiar syntaxes, tools, and paradigms for data transformation and data modelling in R. It allows data pipelines working well with non-distributed data in R to be easily transformed into analogous ones that can process large-scale, distributed data in Apache Spark.
```